frb_codegen/src/generator/dart/func.rs

```diff
@@ -178,6 +184,81 @@ pub(crate) fn generate_api_func(
     }
 }
 
+pub(crate) fn get_api2wire_prefix(
+    api2wire_func: &str,
+    shared_dart_api2wire_funcs: &Option<Acc<String>>,
+    ir_file: &IrFile,
+    ir_type: &IrType,
+    for_dart_basic_file: bool,
```

Comment:
nit: what is a "dart basic file"?

Reply:
according to the def of `Acc`, I think it should be refined to `for_dart_common_file`,fixed in 
096cfd6.